api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageType.java

```diff
@@ -4,7 +4,7 @@
     CREATE_NEW_CARD,
     MAKE_FIRST_PAYMENT,
     MAKE_PAYMENT,
-    SEND_FIRST_EMAIL,
+    SEND_EMAIL,
```

Comment:
This will only work as long as the message is created with the template ID and all required variable values. This might mean that the method that invokes the sending of an email gets a bit too large. I guess we can track that as it happens - we can always reinstate 'send_first_email' and have it amass the required variables then trigger 'send_email' if we need to.

Reply:
I'm going to create a configuration class that stores the template ids and have an enum that the email messages reference instead, when the message context is inflated from the payload we can then lookup the templateId from the enum value. That was my thoughts anyway.

Reply:
Sounds good. It was more knowing which variables to obtain that I was concerned about.